src/app/src/mainwindow.cpp

```diff
@@ -230,6 +233,32 @@ void MainWindow::on_ledIntensityEdit_valueChanged(double value) {
 }
 
 
+/*
+ * Check if drive on windows has sufficient space for acquisition.
+ *
+ * @param drive letter selected in config file
+ * @param fps setting of acquisition
+ * @param duration of acquisition
+ * @param TODO
+ *
+ * @returns boolean true if space is available
+*/
+BOOL available_space_in_drive(std::filesystem::path driver_name, double fps,double duration,double one_image_size  =10000000){
```

Comment:
use `bool` instead of `BOOL`

Reply:
I see bool is the c++ type and BOOL is a Microsoft type that is actually an int.